### PR TITLE
Added a new location and made changes in the timings.

### PR DESCRIPTION
**Pull request by ramavavi**
The following Pull request (PR) addresses: https://github.com/episphere/connectApp/issues/572

Link to the document : https://nih.box.com/s/pdbc23obuwpckgy9wt8kda2bl8h03q2d

Updates to the code:

Added a new location in the HFH My Sample page.
Made a few edits in hours under Henry Ford Medical Center - Livonia location.

### CODE REVIEW

File: js/pages/samples.js [hunk 1]
```diff
@@ -405,6 +405,12 @@ const henry_ford = {
           'Monday to Friday 8:00 a.m. - 4:15 p.m. <br> Saturday 7:00 a.m. - 12:00 p.m.',
           'Parking lots are conveniently located near all hospital entrances for those who do not wish to use valet. Wheelchairs are available at all entrances and handicapped parking is available in each parking area. Valet parking is available at all hospital entrances for $3.00. This service is not available on holidays.',
         ],
+        [
+           'Henry Ford Hospital-West Bloomfield',
+           '6777 West Maple Road<br>West Bloomfield, MI 48322<br>Location: Lab Services First Floor-Bloomfield Reception',
```
Comment (JoeArmani) [inline js/pages/samples.js:410]: Can you confirm this is grammatically correct?
Current: `Location: Lab Services First Floor-Bloomfield Reception.` (hyphen vs dash)
Possibly: `Location: Lab Services First Floor - Bloomfield Reception.`

File: js/pages/samples.js [hunk 2]
```diff
@@ -390,7 +390,7 @@ const henry_ford = {
         [
             'Henry Ford Medical Center - Livonia',
             '29200 Schoolcraft Road<br>Radiology, Floor 1<br>Livonia, MI 48150<br>(734) 523-1050',
-            'Mondays, Tuesdays and Wednesdays 9:00 a.m. - 4:00 p.m.',
+            'Mondays, Tuesdays and Wednesdays 9:30 a.m. - 3:30 p.m.<br> Every 2<sup>nd</sup> and 4<sup>th</sup> Saturday  of the month, 8:00 a.m. to 11:30 a.m.',
```
Comment (jhflorey) [inline js/pages/samples.js:393]: Looks like the documentation describes `Mondays, Tuesdays, and Wednesdays 9:30 a.m. to 3:30 p.m.` ( `to` isntead of  `-`). Please help check it.

File: js/pages/samples.js [hunk 3]
```diff
@@ -405,6 +405,12 @@ const henry_ford = {
           'Monday to Friday 8:00 a.m. - 4:15 p.m. <br> Saturday 7:00 a.m. - 12:00 p.m.',
           'Parking lots are conveniently located near all hospital entrances for those who do not wish to use valet. Wheelchairs are available at all entrances and handicapped parking is available in each parking area. Valet parking is available at all hospital entrances for $3.00. This service is not available on holidays.',
         ],
+        [
+           'Henry Ford Hospital-West Bloomfield',
+           '6777 West Maple Road<br>West Bloomfield, MI 48322<br>Location: Lab Services First Floor-Bloomfield Reception',
+           'Monday to Thursday 6:30 a.m. to 6:00 p.m. <br>Friday 6:30 a.m. to 5:00 p.m.<br>Saturday 7:00 a.m. to 3:00 p.m.',
+           'All self-parking at Henry Ford West Bloomfield Hospital is free. Self-parking areas are available near the Main entrance and at the Henry Ford Cancer entrance. Wheelchairs are available at all entrances. Handicapped-designated parking spaces are located in front of the Main entrance, the West entrance and the Emergency entrance.',
```
Comment (jhflorey) [inline js/pages/samples.js:412]: Looks like that we are missing `Parking Instructions`

**Comment by we-ai** [comment]
Do we have a consistent style ('-' or 'to') for hours?  Tony's recent update used 'to', [here](https://github.com/episphere/connectApp/pull/590).

**Comment by anthonypetersen** [comment]
> 

While I think we can encourage sites to use a specific style, ultimately this is "their" page and they are the ones generating the content.

**Review by anthonypetersen — APPROVED** [review]

**Pull request merged**